database/mssql/scripts/clear-objects.sql

```diff
@@ -1,116 +1,55 @@
-    WHILE @constraint IS NOT NULL
-    BEGIN
-        SELECT @SQL = 'ALTER TABLE [dbo].[' + RTRIM(@name) +'] DROP CONSTRAINT [' + RTRIM(@constraint) +']'
-        EXEC (@SQL)
-        PRINT N'Dropped FK Constraint: ' + @constraint + ' on ' + @name
-        SELECT @constraint = (SELECT TOP 1 CONSTRAINT_NAME FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS WHERE constraint_catalog=DB_NAME() AND CONSTRAINT_TYPE = 'FOREIGN KEY' AND CONSTRAINT_NAME <> @constraint AND TABLE_NAME = @name ORDER BY CONSTRAINT_NAME)
-    END
-SELECT @name = (SELECT TOP 1 TABLE_NAME FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS WHERE constraint_catalog=DB_NAME() AND CONSTRAINT_TYPE = 'FOREIGN KEY' ORDER BY TABLE_NAME)
-END
-GO
-
-/* Drop all Primary Key constraints */
-DECLARE @name VARCHAR(128)
-DECLARE @constraint VARCHAR(254)
-DECLARE @SQL VARCHAR(254)
-
-SELECT @name = (SELECT TOP 1 TABLE_NAME FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS WHERE constraint_catalog=DB_NAME() AND CONSTRAINT_TYPE = 'PRIMARY KEY' ORDER BY TABLE_NAME)
-
-WHILE @name IS NOT NULL
-BEGIN
-    SELECT @constraint = (SELECT TOP 1 CONSTRAINT_NAME FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS WHERE constraint_catalog=DB_NAME() AND CONSTRAINT_TYPE = 'PRIMARY KEY' AND TABLE_NAME = @name ORDER BY CONSTRAINT_NAME)
-    WHILE @constraint is not null
-    BEGIN
-        SELECT @SQL = 'ALTER TABLE [dbo].[' + RTRIM(@name) +'] DROP CONSTRAINT [' + RTRIM(@constraint)+']'
-        EXEC (@SQL)
-        PRINT N'Dropped PK Constraint: ' + @constraint + ' on ' + @name
-        SELECT @constraint = (SELECT TOP 1 CONSTRAINT_NAME FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS WHERE constraint_catalog=DB_NAME() AND CONSTRAINT_TYPE = 'PRIMARY KEY' AND CONSTRAINT_NAME <> @constraint AND TABLE_NAME = @name ORDER BY CONSTRAINT_NAME)
-    END
-SELECT @name = (SELECT TOP 1 TABLE_NAME FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS WHERE constraint_catalog=DB_NAME() AND CONSTRAINT_TYPE = 'PRIMARY KEY' ORDER BY TABLE_NAME)
-END
-GO
-
-/* Drop all tables */
-DECLARE @name VARCHAR(128)
-DECLARE @SQL VARCHAR(254)
-
-SELECT @name = (SELECT TOP 1 [name] FROM sysobjects WHERE [type] = 'U' AND category = 0 ORDER BY [name])
-
-WHILE @name IS NOT NULL
-BEGIN
-    SELECT @SQL = 'DROP TABLE [dbo].[' + RTRIM(@name) +']'
-    EXEC (@SQL)
-    PRINT N'Dropped Table: ' + @name
-    SELECT @name = (SELECT TOP 1 [name] FROM sysobjects WHERE [type] = 'U' AND category = 0 AND [name] > @name ORDER BY [name])
-END
-GO
-
-/* Drop all sequences */
-DECLARE @name VARCHAR(128)
-DECLARE @SQL VARCHAR(254)
-
-SELECT @name = (SELECT TOP 1 [name] FROM sys.sequences ORDER BY [name])
-
-WHILE @name IS NOT NULL
-BEGIN
-    SELECT @SQL = 'DROP SEQUENCE [dbo].[' + RTRIM(@name) +']'
-    EXEC (@SQL)
-    PRINT N'Dropped SEQUENCE: ' + @name
-    SELECT @name = (SELECT TOP 1 [name] FROM  sys.sequences WHERE [name] > @name ORDER BY [name])
-END
-GO
+declare @n char(1)
```

Comment:
script was targetting non-app related schema (etl, pmbc) and then failing to remove those elements, causing the script to fail.

This update attempts to correct that @dfilteau